tests/models/roc_bert/test_modeling_roc_bert.py

```diff
@@ -710,4 +710,4 @@ def test_inference_masked_lm(self):
         # convert to tokens is: ['[CLS]', '巴', '*', '黎', '是', '法', '国', '的', '首', '都', '[SEP]']
         expected_output = torch.tensor([[101, 2349, 115, 7944, 3221, 3791, 1744, 4638, 7674, 6963, 102]])
 
-        self.assertEqual(output_ids, expected_output)
+        assert torch.allclose(output_ids, expected_output)
```

Comment:
We have a mixed usage of `self.assertTrue(torch.allclose` and `assert torch.allclose` in our tests. I would like to propose to keep only one usage, likely the former one. 

Let's see what @sgugger says.

I would also like to explicitly specify `atol` despite there is a default value.

Reply:
Since its `ints` in that case atol would be useless. 
Standardizing our test can be donce, we also sometimes have `torch.testing.assert_allclose` (which btw will be removed in pytorch `1.12`) but should be done in a seperate PR ( a lot more work to do ! )

Reply:
I don't mean to change all files (for the style), but even for this file, I am fine if you prefer to do it in another PR.

Reply:
Let's not spend time trying to change 500 test files unless there is clear impact. Both syntaxes work and are clearly readable :-)